apps/transport/lib/transport_web/templates/feedback/feedback.html.heex

```diff
@@ -0,0 +1,63 @@
+<div class="container">
+  <h2><%= dgettext("feedback", "Leave your feedback") %></h2>
+  <%= form_for :feedback, contact_path(TransportWeb.Endpoint, :send_feedback), [class: "feedback-form no-margin"], fn f -> %>
```

Comment:
Remarque : form_for va être déprécié. J’aurais déjà pu utiliser le composant .form pour prévoir la suite comme ici https://hexdocs.pm/phoenix_html/3.1.0/Phoenix.HTML.Form.html#module-phoenix-liveview-integration 
Voir également https://phoenixonrails.com/blog/you-can-stop-using-form-for-in-phoenix

Reply:
Pour les arguments de form_for, je suis dans le cas "without form_data", voir ici https://hexdocs.pm/phoenix_html/3.1.0/Phoenix.HTML.Form.html#module-without-form-data

Gros avantage : comme aucune variable n’est passée dans le render (pas de @conn, pas de @socket) je peux render ma vue depuis une Phoenix.View ou Phoenix.LiveView sans problème.

Désavantage : on peut pas gérer les cas d’erreur correctement en re-rendant le formulaire avec les input pré-remplis.